packages/datadog-instrumentations/src/iovalkey.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
+
+const {
+  channel,
+  addHook,
+  AsyncResource
+} = require('./helpers/instrument')
+const shimmer = require('../../datadog-shimmer')
+
+const startCh = channel('apm:iovalkey:command:start')
+const finishCh = channel('apm:iovalkey:command:finish')
+const errorCh = channel('apm:iovalkey:command:error')
+
+addHook({ name: 'iovalkey', versions: ['>=0'] }, Redis => {
```

Comment:
```suggestion
addHook({ name: 'iovalkey', versions: ['>=0.0.1'] }, Redis => {
```

I believe that makes the tests pass as well?